flickit-assessment-kit/src/main/java/org/flickit/assessment/kit/adapter/in/rest/assessmentkit/GetKitMinimalInfoResponseDto.java

```diff
@@ -0,0 +1,6 @@
+package org.flickit.assessment.kit.adapter.in.rest.assessmentkit;
+
+import org.flickit.assessment.kit.application.port.in.assessmentkit.GetKitMinimalInfoUseCase.ExpertGroup;
+
+public record GetKitMinimalInfoResponseDto(Long id, String title, ExpertGroup expertGroup) {
```

Comment:
To prevent confusion with the yet-to-be-defined `ExpertGroup` domain class, it is recommended to rename the record `ExpertGroup` (defined in the use case) to `MinimalExpertGroup`.